src/twin/system_info.rs

```diff
@@ -21,15 +74,24 @@ lazy_static! {
 }
 
 #[derive(Default, Serialize)]
-pub struct SystemInfo {
-    #[serde(skip_serializing)]
-    tx_reported_properties: Option<mpsc::Sender<serde_json::Value>>,
+pub struct Info {
     os: serde_json::Value,
     azure_sdk_version: String,
     omnect_device_service_version: String,
     boot_time: Option<String>,
```

Comment:
maybe get os and boot_time from `sysinfo` crate

Reply:
boot_time generated from sysinfo crate and handling moved to from system.rs to system_info.rs

Reply:
used as well os name and version handling from sysinfo crate